Source/Scene/WebMapServiceImageryProvider.js

```diff
@@ -137,25 +148,16 @@ define([
                     }
                 }
             });
-        }
 
-        if (defined(options.clock)) {
-            if (defined(options.parameters)) {
-                options.parameters['time'] = options.clock.currentTime.toString();
+            if (defined(options.clock)){
```

Comment:
This doesn't need to be done in the constructor, you are always setting them in `requestImage`